lib/repo/mapping.go

```diff
@@ -0,0 +1,150 @@
+package repo
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+const missingRepo = "Device repo is missing"
+
+var path string
+
+func init() {
+	//p, err := os.Executable()
+	//if err != nil {
+	//	logrus.Error(err)
+	//}
+
+	path = filepath.Join(baseDir, file)
+}
+
+// DeviceImage contains url, title, username and password which are used after flashing
+type DeviceImage struct {
+	Url   string `json`
```

Comment:
__[go_vet]__
* [vet] struct field tag `json` not compatible with reflect.StructTag.Get: bad syntax for struct tag pair

<!-- Comment by [SideCI](https://sideci.com) -->